connect/cli/plugins/project/extension/helpers.py

```diff
@@ -60,6 +75,22 @@ def bootstrap_extension_project(config, output_dir, overwrite):  # noqa: CCR001
     if not overwrite and os.path.exists(project_dir):
         raise ClickException(f'The destination directory {project_dir} already exists.')
 
+    if save_answers_dir:
+        if os.path.exists(save_answers_dir):
```

Comment:
IMO this check should be done at the very beginning

Reply:
Check in the beginning make sense if we need to terminate wizard, either placing of check doesn't matter imo

Reply:
Yes IMO we should stop if the dest file already exist